### DISCUSSION TITLE
[BUG] el pbupdatetask pisa los campos anteriores de una tasks?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#243] [4. Bug Reports] [answered]
Hola quisiera saber si ahora cuando un pb updatetasks se utliza, pisa los datos anteriores de la tasks?.

Por ejemplo me di cuenta que cuando invito usuarios a una tasks por el bot pbtaskaddeditor en la etapa 1 de una SM y posteriormente cuando en una etapa 2 o 3 utilizo un PBUpdateTasks para cambiar la fecha de término, me borra los editores que estaban incorporado.

Si el comportamiento será de esa manera: Cómo puedo colocar los editores y usuarios que estaban antes en el PBupdatetasks?, Serviría colocar: $VALUE#editors?

Gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG esto ocurre en la nueva versión de PBUpdateTask también?

Según la definición del bot, si el campo editors no está definido, no debería sobre-escribirse este.

Como workaround, lo que mencionas estaría correcto de todas maneras si es que efectivamente hubiese un bug.

**Reply by NicolasPLG** [reply]
hola @guillevalin solo he visto reflejado en el 2.0.0 ese problema, podría ser un QA y ver en detalle lo que pasa y copiar los logs aquí.

**Reply by guillevalin** [reply]
Si te volvió a pasar por favor adjuntar evidencia para revisarlo.

**Reply by NicolasPLG** [reply]
Hice un QA y me di cuenta que era la rutina que pisaba los editores asi que estaría todo bien.

Gracias @guillevalin
